workflow/react-project/src/components/generic/Modal/index.jsx

```diff
@@ -0,0 +1,21 @@
+import './style.less';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ReactDOM from 'react-dom';
+
+const Modal = ({ title, subtitle, content, setModalClosed }) =>
```

Comment:
`Modal` is a blueprint for all your modals, so content can be other components. Pass it as `children`